README.md

```diff
@@ -595,6 +595,32 @@ Untracked files:
 nothing added to commit but untracked files present (use "git add" to track)
 ```
 
+### masterAdm
```

Comment:
@WadeWaldron @rgcase Wanted to keep things as simple as possible in this version. The general idea behind this implementation was to be able to delete exercises:
- Delete exercise_xxx_* folders and just run masterAdm -r
- Add exercises by creating gaps where needed, add new exercise folders at the appropriate location and run masterAdm -r
That would do the trick for adding and deleting exercises.
Can create commands as requested for sure, but wanted to spend time on other things first.